server/prisma/schema.prisma

```diff
@@ -1,54 +1,54 @@
 // This is your Prisma schema file,
 // learn more about it in the docs: https://pris.ly/d/prisma-schema
-
 generator client {
   provider = "prisma-client-js"
 }
 
 datasource db {
   provider = "postgresql"
-  url = "postgresql://dima:1234321@localhost:5432/dima?schema=public"
- // url      = env("DATABASE_URL")
+  url      = env("DATABASE_URL")
 }
 
-enum Status {
-  ACTIVE
-  INACTIVE
+enum Category {
+  clothes
+  food
+  medicine
+  restaurants
+  sport
+  taxi
+  transfers
+  utilities  
 }
```

Comment:
```suggestion
enum Category {
  Clothes = 'Clothes'
  Food = 'Food'
  Medicine = 'Medicine'
  Restaurants = 'Restaurants'
  Sport = 'Sport'
  Taxi = 'Taxi'
  Transfers = 'Transfers'
  Utilities = 'Utilities' 
}
```

1. I use Pascal case for enum in TS https://www.theserverside.com/definition/Pascal-case
2. I would add default value